src/pages/Vaults/Vault/UpdateCollateralModal/index.tsx

```diff
@@ -236,13 +236,11 @@ const UpdateCollateralModal = ({
       return '∞';
     }
 
-    // The vault API returns collateralization as a regular number rather than a percentage
-    const strVaultCollateralizationPercentage = vaultCollateralization?.mul(100).toString();
-    if (Number(strVaultCollateralizationPercentage) > 1000) {
+    if (vaultCollateralization.mul(100).gt(1000)) {
       return `more than ${formatPercentage(1000 / 100, { minimumFractionDigits: 0 })}`;
-    } else {
```

Comment:
Are we ok to lose this extra condition?

Reply:
I see the `else` here as unnecessary. Its cleaner to just return outside of the statement.

Reply:
Ah can see that now - thanks